### ISSUE TITLE
Error in creation tenant super user

### ISSUE THREAD

**Issue by Leggendario12** [django-tenants/django-tenants#517]
Tried this package in creating test sample for a multi-type tenant. Encountered problem with creating superuser for the tenant using this command:

```bash
$python manage.py create_tenant_superuser --username=username --schema=schema_name

error message:
    installed_apps = tenant_types[connection.tenant.tenant_type]['APPS']
AttributeError: 'Object' object has no attribute 'tenant_type'
```

I tried to add a method that will set the tenant_type in the TenantMixin model and result to this error:

```bash
 installed_apps = tenant_types[connection.tenant.tenant_type]['APPS']
KeyError: <bound method UserTypes.tenant_type of <UserTypes: UserTypes object (4)>>
```
I will try to carefully debug it and see what exactly is causing the issue as I am new to this package. I carefully followed the documentation on how to create the necessary settings and configuration.

I tried the example from the repository and it showed the first error when performing the migrate_schema --shared after the makemigration.

**Comment by Leggendario12** [comment]
I added a star to this project as I really like the potential of this project.

**Comment by tomturner** [comment]
I have just tried this and it worked can you confirm you are using the Github dev version and not the release version.

Did you add the extra field in the schema / tenant model called tenant_type that is required when using multi type tenants.

**Comment by Leggendario12** [comment]
Yes, I am using the Github dev version of django-tenants and not the PYPI version which doesn’t contains the dependency module require to configure it based on what is on the documentation.

**Comment by Hudeh** [comment]
having same issues when using the pypi casue i want to use it for production

**Comment by Hudeh** [comment]
python manage.py create_tenant_superuser
get error even after clonning the github example
 installed_apps = tenant_types[connection.tenant.tenant_type]['APPS']
AttributeError: 'Client' object has no attribute 'tenant_type'
